src/styles/cart.css

```diff
@@ -0,0 +1,156 @@
+.maindiv-container {
+  display: flex;
+  justify-content: space-around;
+}
+.cartItem {
+  width: 600px;
+  display: flex;
+  border: 1px solid #4b5563;
+  border-radius: 0.5rem;
+  margin: 1rem;
+  padding: 1rem;
+  flex-wrap: wrap;
+}
+.cartItem > .left > .images {
+  width: 100%;
+  height: 150px;
+  object-fit: contain;

```

Comment:
avoid such complex selectors . Use a good class name instead to target the image